### DISCUSSION TITLE
Test discussions

### DISCUSSION THREAD

**Discussion by ruflin** [ruflin/Elastica#1958] [General]
This is a test to test Github discussions.

**Comment by deguif** [comment]
Nice

**Reply by ruflin** [reply]
I wonder if we should use this to get rid of gitter and other discussion locations. I quite like to have all in one place. Also it allows to transfer Github issues to discussions to make sure Issues is not used for discuss things. WDYT?

**Reply by deguif** [reply]
💯

**Reply by ruflin** [reply]
Sounds like a decision :-) Lets try it out. Should I try to close Gitter? I must confess, I have not touched Gitter in a long time.

**Reply by deguif** [reply]
I never touched it so I would say yes ;)

**Reply by ruflin** [reply]
Left a comment on Gitter to have the upcoming discussions here. Lets see what happens.
